app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -0,0 +1,116 @@
+package org.mozilla.vrbrowser.telemetry;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.os.StrictMode;
+import android.preference.PreferenceManager;
+import android.support.annotation.UiThread;
+import org.mozilla.telemetry.Telemetry;
+import org.mozilla.telemetry.TelemetryHolder;
+import org.mozilla.telemetry.config.TelemetryConfiguration;
+import org.mozilla.telemetry.event.TelemetryEvent;
+import org.mozilla.telemetry.net.HttpURLConnectionTelemetryClient;
+import org.mozilla.telemetry.ping.TelemetryCorePingBuilder;
+import org.mozilla.telemetry.ping.TelemetryMobileEventPingBuilder;
+import org.mozilla.telemetry.schedule.jobscheduler.JobSchedulerTelemetryScheduler;
+import org.mozilla.telemetry.serialize.JSONPingSerializer;
+import org.mozilla.telemetry.storage.FileTelemetryStorage;
+import org.mozilla.vrbrowser.BuildConfig;
+import org.mozilla.vrbrowser.R;
+
+
+public class TelemetryWrapper {
+    private final static String APP_NAME = "FirefoxReality";
+    private final static boolean isEnabledByDefault = true;
+
+    private class Category {
+        private static final String ACTION = "action";
+    }
+
+    private class Method {
+        private static final String FOREGROUND = "foreground";
+        private static final String BACKGROUND = "background";
+    }
+
+    private class Object {
+        private static final String APP = "app";
+    }
+
+    public static boolean isTelemetryEnabled(Context context) {
```

Comment:
We have a utility method to check that: SettingsStore.getInstance(ctx).isTelemetryEnabled()

Reply:
Also we should call setTelemetryEnabled() when the setting is changed in FxR

Reply:
I remove set/getTelemetryEnabled() APIs from TelemetryWrapper. Users only can use set/getTelemetryEnabled() by `SettingsStore.getInstance(ctx)`